[2022-03-15 11:13:47,054] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.test scheduled__2022-03-15T11:18:00+00:00 [queued]>
[2022-03-15 11:13:47,066] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.test scheduled__2022-03-15T11:18:00+00:00 [queued]>
[2022-03-15 11:13:47,066] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 11:13:47,067] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-15 11:13:47,067] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 11:13:47,077] {taskinstance.py:1264} INFO - Executing <Task(BashOperator): test> on 2022-03-15 11:18:00+00:00
[2022-03-15 11:13:47,081] {standard_task_runner.py:52} INFO - Started process 20402 to run task
[2022-03-15 11:13:47,088] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'simple_dag', 'test', 'scheduled__2022-03-15T11:18:00+00:00', '--job-id', '284', '--raw', '--subdir', 'DAGS_FOLDER/simple_dag.py', '--cfg-path', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmpp7fcusvt', '--error-file', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmpe1y95vn3']
[2022-03-15 11:13:47,091] {standard_task_runner.py:77} INFO - Job 284: Subtask test
[2022-03-15 11:13:47,155] {logging_mixin.py:109} INFO - Running <TaskInstance: simple_dag.test scheduled__2022-03-15T11:18:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2022-03-15 11:13:47,207] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=test
AIRFLOW_CTX_EXECUTION_DATE=2022-03-15T11:18:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-15T11:18:00+00:00
[2022-03-15 11:13:47,209] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T
[2022-03-15 11:13:47,211] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "The DAG has ran!"']
[2022-03-15 11:13:47,221] {subprocess.py:85} INFO - Output:
[2022-03-15 11:13:47,227] {subprocess.py:89} INFO - The DAG has ran!
[2022-03-15 11:13:47,229] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-15 11:13:47,251] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=test, execution_date=20220315T111800, start_date=20220315T161347, end_date=20220315T161347
[2022-03-15 11:13:47,308] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-15 11:13:47,341] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-15 11:18:40,373] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.test scheduled__2022-03-15T11:18:00+00:00 [queued]>
[2022-03-15 11:18:40,383] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: simple_dag.test scheduled__2022-03-15T11:18:00+00:00 [queued]>
[2022-03-15 11:18:40,384] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 11:18:40,384] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-03-15 11:18:40,384] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-03-15 11:18:40,393] {taskinstance.py:1264} INFO - Executing <Task(BashOperator): test> on 2022-03-15 11:18:00+00:00
[2022-03-15 11:18:40,397] {standard_task_runner.py:52} INFO - Started process 20585 to run task
[2022-03-15 11:18:40,403] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'simple_dag', 'test', 'scheduled__2022-03-15T11:18:00+00:00', '--job-id', '272', '--raw', '--subdir', 'DAGS_FOLDER/simple_dag.py', '--cfg-path', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmpte0rg09v', '--error-file', '/var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T/tmpinx2_5e_']
[2022-03-15 11:18:40,407] {standard_task_runner.py:77} INFO - Job 272: Subtask test
[2022-03-15 11:18:40,460] {logging_mixin.py:109} INFO - Running <TaskInstance: simple_dag.test scheduled__2022-03-15T11:18:00+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2022-03-15 11:18:40,506] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=test
AIRFLOW_CTX_EXECUTION_DATE=2022-03-15T11:18:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-15T11:18:00+00:00
[2022-03-15 11:18:40,508] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/tl/ljh9dpdd2fnbgmc62qc2b49h0000gn/T
[2022-03-15 11:18:40,509] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "The DAG has ran!"']
[2022-03-15 11:18:40,518] {subprocess.py:85} INFO - Output:
[2022-03-15 11:18:40,524] {subprocess.py:89} INFO - The DAG has ran!
[2022-03-15 11:18:40,525] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-15 11:18:40,548] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=test, execution_date=20220315T111800, start_date=20220315T161840, end_date=20220315T161840
[2022-03-15 11:18:40,587] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-15 11:18:40,625] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
